book_source/02_demos_tutorials_workflows/01_install_pecan.Rmd

```diff
@@ -152,11 +152,12 @@ This will not go into much detail about about how to use Docker -- for more deta
     - PEcAn web interface (running models) -- http://localhost:8000/pecan/ (NOTE: The trailing backslash is necessary.)
     - PEcAn documentation and home page -- http://localhost:8000/
     - BETY web interface -- http://localhost:8000/bety/
-    - File browser (minio) -- http://localhost:8000/minio/
+    - File browser (minio) -- http://localhost:8000/minio/ <!-- ERS: Doesn't work on docker installation -->
```

Comment:
please update w/ instructions from @robkooper on Slack https://pecanproject.slack.com/archives/C8RD9BRLK/p1655147681727739 which should work with fixes in #2936